### ISSUE TITLE
Publish on F-Droid

### ISSUE THREAD

**Issue by Integral-Tech** [dic1911/Momogram#39]
It would be better if Momogram could be published on F-Droid.

**Comment by dic1911** [comment]
No, it has been discussed for many times since the NekoX era already, F-Droid is too slow, and I don't want to go through all the hassle to get us there.

**Comment by devnoname120** [comment]
@Integral-Tech You could ask @IzzySoft to add it to his [IzzyOnDroid](https://apt.izzysoft.de/fdroid) custom repository. 

It's the 3rd most popular F-Droid repository, has 1000+ apps, directly takes the APKs from the GitHub releases of projects, allows self-signed APKs, and the [criteria for inclusion](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Inclusion-Policy) are actually sane and reasonable. You can learn more about it [here](https://apt.izzysoft.de/fdroid/index/info).

**Comment by IzzySoft** [comment]
> It's the 3rd most popular F-Droid repository

Out of curiosity: which is the 2nd?

> and the [criteria for inclusion](https://gitlab.com/IzzyOnDroid/repo/-/wikis/Inclusion-Policy) are actually sane and reasonable

Thanks! They btw also state a per-app size limit of 30 MB; Monogram APKs are twice that size, sorry.

**Comment by Integral-Tech** [comment]
@dic1911 Maybe you could maintain a custom F-Droid repository :)
